packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -307,7 +349,17 @@ export function rebaseChange<TChange>(
 		inverseFromCommit(changeRebaser, commit, mintRevisionTag, true),
 	);
 	inverses.reverse();
-	return rebaseChangeOverChanges(changeRebaser, change, [...inverses, ...targetPath]);
+
+	const telemetryProps = {
+		sourceBranchLength: sourcePath.length,
+		rebaseDistance: targetPath.length,
+		countInCommon: 0, // TODO: Change the value.
```

Comment:
Trying to figure out a way to determine this value without the `sourceSet`.

Reply:
For `rebaseDistance`, I believe the `targetPath.length` is correct as we are appending to the tip of the trunk.

Reply:
This function never drops the rebased commits so returning `0` is correct.